archetypes/business-service-spring-boot-starter/src/main/resources/archetype-resources/pom.xml

```diff
@@ -52,8 +52,9 @@
       <artifactId>spring-boot-starter-provider</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.hibernate</groupId>
+      <groupId>org.hibernate.validator</groupId>
```

Comment:
https://mvnrepository.com/artifact/org.hibernate/hibernate-validator/6.0.2.Final

seems that new groupId is the not correct?